src/Notes/PaymentsRemindMeLater.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * WooCommerce Admin Payment Reminder Me later
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+use Automattic\WooCommerce\Admin\PluginsHelper;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * PaymentsRemindMeLater
+ */
+class PaymentsRemindMeLater {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-admin-payments-remind-me-later';
+
+	/**
+	 * Should this note exist?
+	 */
+	public static function is_applicable() {
```

Comment:
Nice use of this new method!

This is a great example of the direct inverse relationship we typically have behind note creation/deletion and I wonder if this should be the default unless overridden.

cc @jacob-sewell